enhancements/console/dynamic-plugins.md

```diff
@@ -350,6 +350,33 @@ available and the user must refresh their browser to see the changes. (We won't
 refresh the page automatically to avoid possibly losing data if the user is
 entering something into a form or the YAML editor.)
 
+### Localization
+
+Info on how Console is handling i18n is in this [enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/console/internationalization.md).
+
+Console uses [react-i18next](https://github.com/i18next/react-i18next) for i18n,
+and dynamic plugins must use react-i18next as well.
+
+All dynamic plugins must use a single react-i18next [namespace](https://www.i18next.com/principles/namespaces),
+named after the namespace, e.g. for `kubevirt` the filename would be
+`plugin__kubevirt.json`. Localization files need to be served under
+`/locales/{language}/{namespace}.json` path.
+All dynamic plugins are expected to use `plugin__` namespace prefix, e.g.
+`plugin__knative` or `plugin__kubevirt`. The request for the dynamic
+plugin locales will be proxied by console to plugin's `Pod`, using it's `Service`.
+Example request path for `knative` plugin locales will look like:
+`/locales/resource.json?lng=en&ns=plugin__knative'`.
+This request will be proxied to:
+`<service-name>.<service-namespace>.svc.cluster.local:<service-port>/locales/{language}/{namespace}.json`
```

Comment:
I'd keep the plugin example consistent with the code example below, so let's use kubevirt.

```suggestion
plugin localization resources will be proxied by console backend.
For example, the `kubevirt` plugin localization resource
in the `en` language will be requested at path
`/locales/en/plugin__kubevirt.json`
```